### PR TITLE
feat: new icon user speak

### PR DESCRIPTION
**Pull request by fede-dp**
# feat: new icon user speak

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [x ] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :pencil: Checklist

For all PRs:

- [x ] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [ x] I have reviewed my changes.
- [x ] I have added all relevant documentation.
- [ x] I have considered the performance impact of my change.

### CODE REVIEW

**Review by yorbi-dp — CHANGES_REQUESTED** [review]
It's missing the `run build` to create or add the icon in the keywords.json

File: packages/dialtone-icons/src/svg/people/user-speak.svg
Comment (francisrupert) [inline packages/dialtone-icons/src/svg/people/user-speak.svg:1]: Maybe `user-speak`? If only to align to other "user" icons, i.e. https://dialtone.dialpad.com/design/icons/?search=user

Definitely needs `keywords.json` added.

Comment (francisrupert) [reply packages/dialtone-icons/src/svg/people/user-speak.svg:1]: Oh never mind, I'm an idiot. That is the name you used 🙃

**Review by yorbi-dp — APPROVED** [review]
LGTM!

**Comment by fede-dp** [comment]
> LGTM!

was trying to change the icon from marshall proposal that Francis suggested in the figma, i think is ready now!

**Review by braddialpad — APPROVED** [review]

**Pull request merged**